src/applications/ivc-champva/10-10D/config/form.js

```diff
@@ -327,7 +329,10 @@ const formConfig = {
                 updateSchema: (formData, formSchema) => {
                   const fs = formSchema;
                   if (
-                    formData.certifierRelationship.relationshipToVeteran.other
+                    get(
+                      'certifierRelationship.relationshipToVeteran.other',
+                      formData,
+                    )
```

Comment:
Bug fix - when user tried to continue a previously saved form attempt, everything broke because `formData.certifierRelationship` was usually undefined in the saved data. Switching to `get()` so we access it safely and ignore undefined values.